packages/create/src/util.ts

```diff
@@ -20,6 +21,34 @@ export async function replaceStringInFile(
   )
 }
 
+export async function readFileWithRetries(filePath: string, retries: number = 5, delay: number = 1000): Promise<string> {
+  for (let attempt = 1; attempt <= retries; attempt++) {
+    if (fs.existsSync(filePath)) {
+      return await fs.promises.readFile(filePath, "utf-8");
+    }
+    await new Promise(resolve => setTimeout(resolve, attempt * delay));
+  }
+  throw new Error(`${filePath} could not be found after ${retries} attempts`);
+}
+
+export function execCommandExitOnError(
```

Comment:
I did the exec since it spawns a new process, as there might be a bug with spawn.sync because it tries to reuse the same shell. It is the reason I believe 'npm create vite@latest' was failing, but is more of a hunch than 100% confirmed.

I haven't tested on Windows, so likely the better approach would be to see if cross-spawn is needed for Windows support and then decide if the project can just use the simpler standard library function or if the third-party library is needed, which in that case, the execCommandExitOnError should be rewritten to use cross-spawn for compatibility.